lua/autorun/sh_earmuffs_init.lua

```diff
@@ -0,0 +1,35 @@
+CFCEarmuffs = CFCEarmuffs or {}
+
+if SERVER then
+    AddCSLuaFile( "earmuffs/shared/sh_logger.lua" )
+    AddCSLuaFile( "earmuffs/shared/sh_utils.lua" )
+    AddCSLuaFile( "earmuffs/shared/sh_config.lua" )
+
+    AddCSLuaFile( "earmuffs/client/cl_settings.lua" )
+    AddCSLuaFile( "earmuffs/client/cl_networked_sounds.lua" )
+    AddCSLuaFile( "earmuffs/client/cl_combat_sounds.lua" )
+    AddCSLuaFile( "earmuffs/client/cl_sweps.lua" )
+    AddCSLuaFile( "earmuffs/client/cl_tool_menu.lua" )
```

Comment:
Seems all of these files call `AddCSLuaFile` themselves, they're being added twice.
I don't know where you want those adds should be, though it would make sense for them to be in this init file.

Reply:
Yeah I can remove the ones from within the shared files - luckily it doesn't actually send it to the client twice